src/SystemCommands-RefactoringSupport/TRefactoringCommandSupport.trait.st

```diff
@@ -30,8 +30,9 @@ TRefactoringCommandSupport >> initializeDefaultOptionsOf: refactoring [
 { #category : 'initialization' }
 TRefactoringCommandSupport >> setUpModelFromContext: aContext [
 	| browser scope |
+	
 	browser := aContext browser.
 	scope := browser defaultNavigationScope.
-	scope isCurrentImage ifTrue: [ scope := RefactoringSettings scopeRefactoring ].
+	scope isCurrentImage ifTrue: [ scope := ScopesManager defaultScope ].
```

Comment:
@carolahp @tesonep This causes a MessageNotUnderstood (“Instance of ScopesManager class did not understand #defaultScope”) in Pharo 12 build 1434. It probably needs to be:

```smalltalk
scope isCurrentImage ifTrue: [ scope := ScopesManager singleInstance defaultScope ].
```

Reply:
I see now that this is because of the [dependency on NewTools pull request #718](https://github.com/pharo-spec/NewTools/pull/718/files#diff-016cac8f9b8c115c4300e73e84d59148a0a15ce74237db611da7d47d51a1994aR80-R83) (which has not been merged yet).